doc/man3/PKCS7_sign_add_signer.pod

```diff
@@ -30,16 +30,31 @@ Unless the B<PKCS7_REUSE_DIGEST> flag is set the returned B<PKCS7> structure
 is not complete and must be finalized either by streaming (if applicable) or
 a call to PKCS7_final().
 
+PKCS7_add_certificate() adds to the B<PKCS7> structure I<p7> the certificate
+I<cert>, which may be an end-entity (signer) certificate
+or a CA certificate useful for chain building.
+This is done internally by L<PKCS7_sign_ex(3)> and similar signing functions
+but can also be called directly to add a further such certificate.
+It may be used before calling L<PKCS7_verify(3)>
+to provide any missing certificate(s) needed for chain building
+or to provide a fallback signer certificate.
```

Comment:
Are these really *fallback*, IIRC they are used in preference to those included in the message?

Reply:
`PKCS7_add_certificate()` (as well as `CMS_add0_cert()` and `CMS_add1_cert()`) add certs to the message,
and these are treated as fallback,
in comparison to those in the `certs` parameter of `PKCS7_verify()`, which are preferred.